include/sacn/cpp/source.h

```diff
@@ -287,8 +293,8 @@ inline etcpal::Error Source::Startup(const Settings& settings)
  */
 inline void Source::Shutdown()
 {
-  sacn_source_destroy(handle_);
-  handle_ = kInvalidHandle;
+  sacn_source_destroy(handle_.value());
+  handle_.SetValue(SACN_SOURCE_INVALID);
```

Comment:
You can use handle_.Clear() as a shortcut here

Reply:
https://github.com/ETCLabs/sACN/pull/45/commits/ebef5bebf5270959fa56c89c6cb1887fdd606c83